deployment/volumes/volume-snapshots/snapshot.yaml

```diff
@@ -2,7 +2,7 @@
   apiVersion: snapshot.storage.k8s.io/v1alpha1
   kind: VolumeSnapshot
   metadata:
-    name: uploads-snapshot
+    name: YYYY-MM-DD-uploads-snapshot
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-04-08T13:19:35Z" title="Wednesday, April 8th 2020, 3:19:35 pm +02:00">Apr 8, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -2,7 +2,7 @@
   apiVersion: snapshot.storage.k8s.io/v1alpha1
   kind: VolumeSnapshot
   metadata:
-    name: uploads-snapshot
+    name: YYYY-MM-DD-uploads-snapshot
```


can you still `apply` this file? or would it need to be updated with the year, month and day?

Reply:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-04-09T10:54:39Z" title="Thursday, April 9th 2020, 12:54:39 pm +02:00">Apr 9, 2020</time>_

---


Each snapshot has a timestamp when it has been created, so is this necessary? For uniqueness, you could just ask the user to pick a different name

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-04-13T07:25:05Z" title="Monday, April 13th 2020, 9:25:05 am +02:00">Apr 13, 2020</time>_

---


@Tirokk brought up that it makes it easier to see what day it was created by prepending the date. With the timestamp, you can easily see how many days ago it was created, but why not make it obvious what day, month, year? Also, this is an improvement over `uploads-claim-{n}`, don't you think?

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2020-04-15T09:17:52Z" title="Wednesday, April 15th 2020, 11:17:52 am +02:00">Apr 15, 2020</time>_

---


This have to be edited before one `apply` for sure.